contracts/BodhiToken.sol

```diff
@@ -1,9 +1,9 @@
 pragma solidity ^0.4.10;
 
 import './StandardToken.sol';
-import './Ownable.sol';
+import './Pausable.sol';
 
-contract BodhiToken is StandardToken, Ownable {
+contract BodhiToken is StandardToken, Pausable {
```

Comment:
Let's keep the current contract as it is. Try not to add new feature, only make sure it is secure.